credentials/README.md

```diff
@@ -64,21 +64,21 @@ which provide ACME service.
       ```diff
        # the last few lines in input.yaml
        certificates:
-      -  pre_issued:
+      -  !pre_issued:
```

Comment:
are the ! in front of some of the fields intentional? Nothing mentioned in the description of the PR

Reply:
Yes, this is intentional. PR description is now modified to mention this.